tests/testthat/test-spaces_left_parentheses_linter.R

```diff
@@ -77,3 +77,11 @@ test_that("returns the correct linting", {
 
   expect_lint("a <- -(b)", NULL, spaces_left_parentheses_linter)
 })
+
+test_that("doesn't produce a warning", {
+  expect_silent(
+    lint(
+      system.file("example/complexity.R", package = "lintr")
+    )
+  )
+})
```

Comment:
trailing newline

(should we add that as a default linter btw?)

Reply:
It's already part of the default linters?
cf. `zzz.R#L95`

Reply:
Sorry, i meant a linter for the terminal newline, which I just realized, isn't implemented yet! proceed...